src/BizHawk.Client.EmuHawk/tools/TAStudio/TAStudio.cs

```diff
@@ -600,8 +600,9 @@ private void StartNewTasMovie()
 				Config.DefaultAuthor);
 
 			SetTasMovieCallbacks(tasMovie);
-			tasMovie.ClearChanges(); // Don't ask to save changes here.
-			tasMovie.Save();
+			// QueueNewMovie will write to disk because Changes == true, and RunQueuedMovie will update MovieSession props

```

Comment:
This comment is wrong, as movie.Mode is MovieMode.Inactive currently, preventing the save from actually occurring.